test/e2e/01-install.sh

```diff
@@ -66,13 +68,17 @@ set -e
 kubectl create secret tls -n tekton-pipelines tekton-results-tls --cert="${SSL_CERT_PATH}/tekton-results-cert.pem" --key="${SSL_CERT_PATH}/tekton-results-key.pem" || true
 
 echo "Installing Tekton Results..."
-kubectl kustomize "${ROOT}/test/e2e/kustomize" | ko apply -f -
+if [[ "$(uname -a)" =~ "ARM64" && "$(uname -m)" =~ "x86_64" ]]; then
+    extra_ko_params="linux/arm64"
+fi
```

Comment:
I don't see `extra_ko_params` being initialized outside of this if block, leading to potential problems.

Does it not make sense to use the go OS/ARCH env values?

```suggestion
exra_ko_params="$(go env GOOS)/$(go env GOARCH)"
```

Reply:
Agree, it does.

Reply:
> Does it not make sense to use the go OS/ARCH env values?

As ko only builds base images in format
<img width="749" alt="image" src="https://user-images.githubusercontent.com/51620925/215644056-ab4d2e47-726c-41b6-93b7-17efd2672a6e.png">
I think using `exra_ko_params="linux/$(go env GOARCH)"` would make sense. WDYT? Because on mac it will give `Darwin/arm64`

Reply:
Done address the changes. PTAL 🙂

Reply:
/lgtm